parity-crypto/src/hmac.rs

```diff
@@ -86,3 +86,51 @@ impl VerifyKey<digest::Sha512> {
 pub fn verify<T>(k: &VerifyKey<T>, data: &[u8], sig: &[u8]) -> bool {
 	hmac::verify(&k.0, data, sig).is_ok()
 }
+
+
+#[cfg(test)]
+::tests_hmac!();
+
+#[cfg(test)]
+#[macro_export]
+macro_rules! tests_hmac {
```

Comment:
It looks like we don't need this macro now?

Also by the way, you can create a separate test file and use `mod` with `path` attribute to include it multiple times. That should make everyone's formatter much happier.

Reply:
Thanks for the 'path' siggestion, it will so much cleaner.